app/keepers/keepers.go

```diff
@@ -350,7 +364,10 @@ func (ak *SecretAppKeepers) InitCustomKeepers(
 		appCodec,
 		ak.keys[ibctransfertypes.StoreKey],
 		ak.GetSubspace(ibctransfertypes.ModuleName),
-		ak.PacketForwardKeeper,
+		// todo: verify the following: the transfer keeper does not need to know about packet forward keeper, because
+		//  we don't want to go through forward module if the packets originated in this chain.
+		// todo: verify the following: we want fees for the transfer app (it previously didn't have)
+		ak.IbcFeeKeeper, // integrate fee channel with transfer
```

Comment:
Osmosis doesn't bother with PacketForward on the Send direction, since PacketForward can become active only on receive direction. But I returned it to be present in both directions of the stack after I merged Assaf's Ibc-hooks, for simplicity